common/views/components/SearchForm/SearchForm.tsx

```diff
@@ -76,11 +74,7 @@ const SearchForm = ({
         form={`search-form-${searchCategory}`}
         placeholder={
           searchLabelText[
-            searchCategory !== 'overview'
-              ? searchCategory
-              : allSearch
-                ? 'overviewAllSearch'
-                : 'overview'
+            searchCategory !== 'overview' ? searchCategory : 'overview'
```

Comment:
Could this be just `searchCategory`? Might be missing something.

Reply:
I think you're right. Will update